src/metal/engine.ts

```diff
@@ -16,6 +16,7 @@ export class Engine implements EngineInterface {
   private reads: Array<Function> = [];
   private work: Array<Function> = [];
   private running: Boolean = false;
+  private skipCounter: number = 4;
```

Comment:
can you create a `const` at top of file named something like `NUM_SKIPPED_FRAMES` or something like that?